lib/sass/script/value/callable.rb

```diff
@@ -0,0 +1,48 @@
+module Sass::Script::Value
+  # A SassScript object representing a null value.
+  class Callable < Base
+    # Constructs a Callable value for use in SassScript.
+    #
+    # @param callable [Sass::Callable] The callable to be used when the
+    # callable is called.
+    def initialize(callable)
+      super(callable.dup)
```

Comment:
I wish we'd freeze objects that we don't intend to mutate. It's very hard to be sure whether a reference should be dup'd or not.